decidim-core/app/packs/stylesheets/decidim/_layout.scss

```
@@ -11,7 +11,7 @@
 }
 
 .layout-1col {
-  @apply container grid grid-rows-[max-content] grid-cols-12 auto-rows-max mb-12 grow;
+  @apply container grid grid-rows-[max-content] grid-cols-12 auto-rows-max mb-12 grow max-w-full;
```

I can't see any difference before and after this change, so I think it's better to drop it 

```suggestion
  @apply container grid grid-rows-[max-content] grid-cols-12 auto-rows-max mb-12 grow;
```

---

@andreslucena this affects the width of the container allowing the buttons vertically sit on top of each other to within a container too narrow affecting proposed changes.

---

Please let me resolve myself the conversations, specially if the actionable thing that I mentioned wasn't applied :) 

Bear in mind that this particular class (`layout-1col`) is used by a partial (`layout_center`) that's being used by 53 pages, and to be honest I don't know if any of these pages is expecting that the `max-w-full` isn't being set. 

If this change is only necessary for these pages (i.e. login and maybe create an account) then we should add the class to the div itself.

---

I think there's no need to pass it as a local variable to the partial, as the same class should be directly applies to the code of the partial itself. As the "variable" is always the same, then it isn't a variable, right?

Also now it isn't doing anything as: 

1. You're still setting the class in this file (`_layou.scss`)
2. Inside of the partial you're not calling to the variable (i.e. `<%= class %>`)

But as I say, as the class is always the same its just better to add the `max-w-fuill` class to the partial itself.

---

@andreslucena updated changes to call the ```max-w-full``` class to the partial itself while removing directly from the (```_layout.scss```) file.